src/bindings/js/node/src/model_wrap.cpp

```diff
@@ -117,3 +119,30 @@ Napi::Value ModelWrap::get_outputs(const Napi::CallbackInfo& info) {
 
     return js_outputs;
 }
+
+Napi::Value ModelWrap::set_friendly_name(const Napi::CallbackInfo& info) {
+    Napi::Env env = info.Env();
+    if (info.Length() != 1 || !info[0].IsString()) {
+        reportError(env , "Expected a single string argument for the friendly name" );
```

Comment:
Make sure that you are using openvino [clang format](https://github.com/openvinotoolkit/openvino/blob/master/src/.clang-format) to format your code
```suggestion
        reportError(env , "Expected a single string argument for the friendly name");
```